rolling/distribution.yaml

```diff
@@ -3068,7 +3068,7 @@ repositories:
       tags:
         release: release/rolling/{package}/{version}
       url: https://github.com/ros2-gbp/rosidl_typesupport-release.git
-      version: 1.4.0-1
+      version: 1.3.0-1
```

Comment:
This change is already in main and cannot be just rolled back. It needs a higher version released with the old content.

Reply:
If these releases are in main then I'd just as soon not revert them at all since they're clearly not causing the regressions.

Reply:
Hmm, let's have a look at those debians. Dependency exportation changes were meant to be used together.